mesonbuild/dependencies/misc.py

```diff
@@ -213,7 +217,7 @@ def __init__(self, environment, kwargs):
 
         if not self.is_found and mesonlib.is_windows():
             # only Intel Fortran compiler is compatible with Microsoft MPI at this time.
-            if language == 'fortran' and environment.detect_fortran_compiler(self.for_machine).name_string() != 'intel':
+            if language == 'fortran' and environment.detect_fortran_compiler(self.for_machine).name_string() != 'intel-ci':
```

Comment:
that should be `intel-cl` not `intel-ci`, right?

Reply:
Thanks! corrected in f990c3e